### PR TITLE
QuestionPrompt CSS setup

### PR DESCRIPTION
**Pull request by chethanagopinath**
Looks like this

Screen Shot 2022-06-28 at 1 49 35 PM

Would be cool if we could create an issue to clean up `questionsData` and replace with actual questions/choice and deal with some formatting that comes with it

### CODE REVIEW

File: src/components/Question.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    display: block;
+    border: none;
+    background-color: #FFFFFF;
+    color: #555555;
+    padding: 0px;
+}
+
+.option:hover {
+    background-color: #FFFFFF;
+}
```
Comment (meganweijiang) [inline src/components/Question.css:19]: Let's do 
```
.option:hover {
    background-color: #FFFFFF;
    color: #C82B2B;
}
```

Comment (meganweijiang) [reply src/components/Question.css:19]: also I forgot to do this, but if you can update the `button` styling in `App.css` to
```
button {
  font-size: 36px;
  background-color: #C82B2B;
  border: 3px solid #555555;
  color: white;
  padding: 10px 20px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  margin: 40px;
  border-radius: 8px;
  cursor: pointer;
}
```

that'd be great! I missed the `cursor: pointer` part 😆

**Review by meganweijiang — COMMENTED** [review]
Looks amazing!!! Just had a couple comments

**Review by meganweijiang — COMMENTED** [review]
oh no now there are conflicts 😅

**Comment by chethana-gopinath** [comment]
oooops 🙈 let me see

**Review by meganweijiang — APPROVED** [review]
🎉

File: src/components/Question.css (hunk 1 above)
Comment (chethana-gopinath) [reply src/components/Question.css]: looks really cool with the red highlight on hover + pointer 💯

**Review by jraigosaOC — APPROVED** [review]

**Pull request merged**